browser/brave_features_internal_names.h

```diff
@@ -11,6 +11,7 @@
 
 inline constexpr char kPlaylistFeatureInternalName[] = "playlist";
 inline constexpr char kPlaylistFakeUAFeatureInternalName[] = "playlist-fake-ua";
+inline constexpr char kSplitViewFeatureInternalName[] = "brave-split-view";
```

Comment:
Should this one also be under `#if !BUILDFLAG(IS_ANDROID)`?

Reply:
That'd be better! Thanks.